### PR TITLE
Feature/KX-78-Modify-Columns-Definitions

### PR DESCRIPTION
**Pull request by golnazsaraji**

### CODE REVIEW

**Review by AlbertoPedalino — CHANGES_REQUESTED** [review]
image

**Review by AlbertoPedalino — CHANGES_REQUESTED** [review]
remove controls when a group by is active, and reduce the distance between icons

**Review by lbordino — APPROVED** [review]
Can you explain the sizing strategy?

File: client/src/components/list/List.tsx [hunk 1]
```diff
@@ -179,12 +184,13 @@ function List(props: ListProps) {
...
   };
 
-  function onGridReady() {
+  function onGridReady(params: any) {
     const allColumnIds: string[] = [];
     gridRef.current!.api!.getAllGridColumns()!.forEach((column) => {
       allColumnIds.push(column.getId());
     });
     gridRef.current!.api!.autoSizeColumns(allColumnIds, false);
+    params.api.sizeColumnsToFit();
```
Comment (lbordino) [inline client/src/components/list/List.tsx:193]: why are we perform multiple sizing procedure?

**Review by lbordino — COMMENTED** [review]
Can you explain the sizing strategy?

File: client/src/components/list/List.tsx (hunk 1 above)
Comment (golnazsaraji) [reply client/src/components/list/List.tsx:193]: one is for fitting the content inside each column. another is for fitting the columns on the container. As I know

**Review by GiuseppeArbore — APPROVED** [review]

**Pull request merged**